app/models/intake.rb

```diff
@@ -245,10 +245,14 @@ def any_students?
       dependents.where(was_student: "yes" ).any?
   end
 
+  def spouse_name_or_placeholder
+    spouse.present? ? spouse.full_name : "Your spouse"
```

Comment:
fun rails tip: this exists to handle this pattern, should you so choose:

```ruby
spouse.full_name.presence || "Your Spouse"
```

Reply:
@tdooner that works for spouse presence or just full name presence?

Reply:
looks like that's for full_name. I changed it to

```ruby
spouse&.full_name || "Your spouse"
```

Reply:
Sorry yeah, I misread this. That looks good.